install-test.sh

```diff
@@ -0,0 +1,3 @@
+sudo python setup.py develop
```

Comment:
Why does install-test also do a ```sudo python setup.py develop```?  Does it require nengo_gui to be installed in this way?